packages/devtools_app/lib/src/code_size/code_size_controller.dart

```diff
@@ -131,9 +140,14 @@ class CodeSizeController {
 
     final diffMap = buildComparisonTreemap(oldFile.data, newFile.data);
     diffMap['n'] = 'Root';
-    final newRoot = generateDiffTree(diffMap);
 
-    changeDiffRoot(newRoot);
+    _combinedDiffTreeRoot = generateDiffTree(diffMap, DiffTreeType.combined);
+    _increasedDiffTreeRoot =
+        generateDiffTree(diffMap, DiffTreeType.increaseOnly);
+    _decreasedDiffTreeRoot =
+        generateDiffTree(diffMap, DiffTreeType.decreaseOnly);
+
+    changeDiffRoot(_combinedDiffTreeRoot);
```

Comment:
change to the active diff tree root. You may need to make `activeDiffTreeRoot` a getter that switches on the active tree type. As a user if I want to look at increase only for one diff, and then I clear and load a second diff, I don't want the default so switch back to combined, I was to see the diff type that I left off at.

Reply:
Done